### PR TITLE
Some improvements to the menu

### PR DESCRIPTION
**Pull request by mueller-ma**
This commit will
* Add several icons with different resolutions
* Translate "Voice Recognizition"
* Disable NFC Menu entry, if no NFC is available
* Reorder menu entries
* Apply material typing to strings

Same as #350 

Signed-off-by: mueller-ma <mueller-ma@users.noreply.github.com>

### CODE REVIEW

**Comment by mueller-ma** [comment]
The commit message is wrong: It never shows nfc in the action bar

File: mobile/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -102,6 +102,9 @@
     <string name="mtm_notification">Certificate verification</string>
     <string name="settings_openhab_none">none</string>
     <!-- theme -->
+    <string name="theme_value_light" translatable="false">light</string>
+    <string name="theme_value_dark" translatable="false">dark</string>
+    <string name="theme_value_black" translatable="false">black</string>
     <string name="theme_value_light">light</string>
```
Comment (digitaldan) [inline mobile/src/main/res/values/strings.xml]: Should this be removed?  They are listed here twice now.

**Comment by digitaldan** [comment]
Hi, this LGTM except for the one comment and the merge conflict, I can merge when those are resolved.

**Comment by digitaldan** [comment]
Also I noticed settings was moved from the first entry to the last, even though its probably the most used menu, is there a thought why?

**Comment by mueller-ma** [comment]
I always expect that the settings is one of the last points

**Comment by digitaldan** [comment]
@mueller-ma unfortunately the string files seem to be in constant change with every PR, can you fix the conflicts?

**Pull request merged**